packages/realm/bindgen/js_spec.yml

```diff
@@ -7,7 +7,7 @@ classes:
   JsPlatformHelpers:
     abstract: true
     staticMethods:
-      # set_default_realm_file_directory: '(dir: std::string)' # currently only used via android JNI
+      set_default_realm_file_directory: '(dir: std::string)'
```

Comment:
When rebasing, you'll have to add `set_default_realm_file_directory` [here](https://github.com/realm/realm-js/blob/569fbb82c2cbb036b682d95234d1a48fa0304254/packages/realm/bindgen/js_opt_in_spec.yml#L177) as well 👍